.changes/unreleased/Added-20240916-122324.yaml

```
@@ -0,0 +1,6 @@
+kind: Added
+body: A Python project's tool.uv index-url setting will be respected and used as the index-url for dagger's codegen as well as project package retrieval via uv.
+time: 2024-09-16T12:23:24.696966201-05:00
+custom:
+    Author: Clif Houck
```

Should be your GH username, not name. 
```suggestion
    Author: ClifHouck
```

---

Addressed in https://github.com/dagger/dagger/pull/8431/commits/f290f71d11f6cf7fc6fdc883e57602bb53a695e5

---

Not commited? 🙂

---

Whoops, fixed this time.